README.md

```diff
@@ -1 +1,21 @@
-# semaphore-stellar-docs
\ No newline at end of file
+# semaphore-stellar-docs
+
+Welcome to the Semaphore Stellar documentation repository. This repository serves as the central hub for all technical documentation and research related to the Semaphore Implementation in Stellar.
+
+## Repository Organization
+
+The repository is structured to make finding information as straightforward as possible. All technical documentation is available in the root directory, providing immediate access to essential implementation details, guides, and specifications.
+
+For research-related content, we maintain a dedicated `research` folder. This folder contains all research files and related materials. When contributing new research, please ensure your files are added to this folder to maintain our organizational structure.
+
+## Community Support
+
+We have an active community ready to help! For additional support or to engage with our team, you can join our Telegram groups:
+
+### Semaphore Stellar
+- If you're interested in discussing with Semaphore Stellar maintainers directly
+- Join: https://t.me/+-9623JNgGjEyNzI5
+
+### Semaphore Community
```

Comment:
Try to add a comment that we're not linked to Semaphore Core team but that the group can be used for direct support.